test/testfiles/testFile.txt

```diff
@@ -0,0 +1,3 @@
+Hello world,
+i'm a testfile.
```

Comment:
"I'm". Can't have typos in our test files.

Reply:
`test file` 😉